website/docs/discover.md

```diff
@@ -0,0 +1,176 @@
+# Discover Podman Desktop in 10 minutes
+
+Podman Desktop is the central place to work with containers as a developer.
+
+The main purpose of Podman Desktop is to use the Docker, Podman and Kubernetes APIs
+to communicate with containers engines and Kubernetes clusters to provide you a graphical
+and unified interface across all these resources.
+
+It also provides extensions to help you install and configure these resources:
+install the Podman engine locally, install Kubernetes clusters locally,
+configure access to remote Kubernetes clusters, configure access to images registries,
+install various CLIs, and more.
+
+## Architecture
+
+The containers technology is a Linux kernel's technology. On a Linux system, you will be able to run Docker and/or Podman
+directly on your system. On other systems like Windows and macOS, you will need to run a Linux instance inside a virtual machine
+and run Docker or podman on this Linux instance, then access them from your system.
+
+### Containers on Linux systems
+
+On Linux systems, Docker is running as a daemon and creates a socket so clients (including the docker cli) can access it through its API.
+The daemon receives commands from the clients, and executes them to build containers, store images, etc.
+
+Podman, on the other hand, is daemonless. This means that the clients (including the podman cli) access directly
+the system resources (containers, filesystem, etc), using a Podman library, to do these operations.
+
+Podman can also run on a Docker-compatibility mode. When doing this, you cannot run Docker and Podman simultaneously anymore.
```

Comment:
FYI you can still run side by side podman and docker.
Each one can use its socket ( for example you have 'docker context' command)